src/index.ts

```diff
@@ -1,61 +1,25 @@
-import { Reshuffle, BaseConnector, EventConfiguration } from 'reshuffle-base-connector'
-
-export interface _CONNECTOR_NAME_ConnectorConfigOptions {
-  var1: string
-  // ...
-}
-
-export interface _CONNECTOR_NAME_ConnectorEventOptions {
-  option1?: string
-  // ...
-}
-
-export default class _CONNECTOR_NAME_Connector extends BaseConnector<
-  _CONNECTOR_NAME_ConnectorConfigOptions,
-  _CONNECTOR_NAME_ConnectorEventOptions
-> {
-  // Your class variables
-  var1: string
-
-  constructor(app: Reshuffle, options?: _CONNECTOR_NAME_ConnectorConfigOptions, id?: string) {
-    super(app, options, id)
-    this.var1 = options?.var1 || 'initial value'
-    // ...
-  }
-
-  onStart(): void {
-    // If you need to do something specific on start, otherwise remove this function
-  }
-
-  onStop(): void {
-    // If you need to do something specific on stop, otherwise remove this function
-  }
-
-  // Your events
-  on(
-    options: _CONNECTOR_NAME_ConnectorEventOptions,
-    handler: any,
-    eventId: string,
-  ): EventConfiguration {
-    if (!eventId) {
-      eventId = `_CONNECTOR_NAME_/${options.option1}/${this.id}`
-    }
-    const event = new EventConfiguration(eventId, this, options)
-    this.eventConfigurations[event.id] = event
-
-    this.app.when(event, handler)
-
-    return event
+import mysql2 from 'mysql2'
+import mysqlPromise from 'mysql2/promise'
+import { Reshuffle, BaseConnector } from 'reshuffle-base-connector'
+
+export default class MySQLConnector extends BaseConnector<null, null> {
+  private client?: any
+  private clientPromise?: any
+
+  constructor(app: Reshuffle, id?: string) {
```

Comment:
Normally in the connector, we encapsulate the options. I suppose in this case these will be the connection details (host, db, username, password, maybe port as well?)
Are we likely to expose an sdk that lets the user create multiple connections to multiple databases, or are we doing one connector instance per db?